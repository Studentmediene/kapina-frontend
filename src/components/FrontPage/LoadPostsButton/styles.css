.loadMore {
  font-size: 22px;
  margin-top: 1em;
  border-width: 0;
}

.loadMore:hover {
  cursor: pointer;
}

.arrow {
  margin-left: 6px;
  width: 20px;
}
