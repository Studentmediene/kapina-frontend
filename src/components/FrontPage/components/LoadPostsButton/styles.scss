.loadMore {
  font-size: 22px;
  margin-top: 1em;
  border-width: 0;
  cursor: pointer;
}

.arrow {
  margin-left: 6px;
  width: 20px;
}
