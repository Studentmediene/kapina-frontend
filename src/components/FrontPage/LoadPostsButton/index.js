import React from 'react';
import PropTypes from 'prop-types';

import styles from './styles.css';
import arrow from './assets/arrow_down.svg';

const LoadPostsButton = props => (
  <button className={styles.loadMore} onClick={props.loadPosts}>
    Last inn flere artikler
    <img src={arrow} alt="Arrow" className={styles.arrow} />
  </button>
);

LoadPostsButton.propTypes = {
  loadPosts: PropTypes.func.isRequired,
};

export default LoadPostsButton;
