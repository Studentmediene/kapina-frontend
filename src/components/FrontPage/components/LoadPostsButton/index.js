import React from 'react';
import PropTypes from 'prop-types';

import styles from './styles.scss';
import arrow from './assets/arrow_down.svg';

const LoadPostsButton = props => (
  <button className={styles.loadMore} onClick={props.loadPosts}>
    Last inn flere artikler
    <img src={arrow} alt="" className={styles.arrow} />
  </button>
);

LoadPostsButton.propTypes = {
  loadPosts: PropTypes.func.isRequired,
};

export default LoadPostsButton;
