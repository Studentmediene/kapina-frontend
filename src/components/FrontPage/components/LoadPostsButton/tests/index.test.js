import React from 'react';
import { shallow } from 'enzyme';

import LoadPostsButton from '../';

describe('<LoadPostsButton />', () => {
  const props = {
    loadPosts: jest.fn(),
  };

  it('renders correctly', () => {
    const tree = shallow(<LoadPostsButton {...props} />);
    expect(tree).toMatchSnapshot();
  });
});
