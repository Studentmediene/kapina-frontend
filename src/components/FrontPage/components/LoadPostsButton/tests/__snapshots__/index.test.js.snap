// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoadPostsButton /> renders correctly 1`] = `
<button
  className="loadMore"
  onClick={[MockFunction]}
>
  Last inn flere artikler
  <img
    alt="Pointing arrow"
    className="arrow"
    src="/assets/arrow_down.svg"
  />
</button>
`;
